pkg/hintrunner/zero/zerohint_signature.go

```diff
@@ -87,7 +98,15 @@ func createVerifyZeroHinter(resolver hintReferenceResolver) (hinter.Hinter, erro
 	return newVerifyZeroHint(val, q), nil
 }
 
-func newVerifyECDSASignatureHinter(ecdsaPtr, signature_r, signature_s hinter.ResOperander) hinter.Hinter {
+// VerifyECDSASignature hint adds an ECDSA signature to a given address
```

Comment:
adds to a given address, or writes to a given address?

Reply:
writes yes, thanks!